docs/pull_request_template.md

```diff
@@ -1,52 +1,29 @@
-# Pull Request Template
+This PR addresses #(insert-number-here)
 
-## Issue Overview
-
-This PR addresses #issue
-
-- [ ] This change addresses the issue in full
-- [ ] This change addresses only certain aspects of the issue
-- [ ] This change is a dependency for another issue
-- [ ] This change has a dependency from another issue
+- [ ] Addresses the issue in full
+- [ ] Addresses only certain aspects of the issue
 
 ## Description
 
-Please include a summary of the change and which issue(s) is addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+Please include a summary of the change and which issue(s) is addressed.
 
 ## How Can This Be Tested/Reviewed?
 
-Provide instructions so we can review.
-
-Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.
+Provide instructions so we can review, including any needed configuration, and the test cases that need to be QAd.
 
-## Checklist:
+## Author Checklist:
 
-- [ ] My code follows the style guidelines of this project
-- [ ] I have added QA notes to the issue with applicable URLs
-- [ ] I have performed a self-review of my own code
-- [ ] I have reviewed the changes in a desktop view
-- [ ] I have reviewed the changes in a mobile view
-- [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have made corresponding changes to the documentation
-- [ ] My changes generate no new warnings
-- [ ] I have added tests that prove my fix is effective or that my feature works
-- [ ] New and existing unit tests pass locally with my changes
-- [ ] Any dependent changes have been merged and published in downstream modules
-- [ ] I have assigned reviewers
-- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
-- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
-- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits
+- [ ] Added QA notes to the issue with applicable URLs
+- [ ] Reviewed in a desktop view
+- [ ] Reviewed in a mobile view
+- [ ] Reviewed considering accessibility
+- [ ] Added tests covering the changes
+- [ ] Made corresponding changes to the documentation
```

Comment:
sorry this is a late review here but can we also include: `I have run yarn generate:client` it catches me ALL the time forgetting to do it

Reply:
Yes absolutely! Updated